sys-kernel/gentoo-kernel/gentoo-kernel-5.9.8.ebuild

```diff
@@ -81,54 +81,17 @@ src_prepare() {
 			;;
 	esac
 
-	local config_tweaks=(
-		# replace (none) with gentoo
-		-e 's:^CONFIG_DEFAULT_HOSTNAME=:&"gentoo":'
-		# we do support x32
-		-e '/CONFIG_X86_X32/s:.*:CONFIG_X86_X32=y:'
-		# disable signatures
-		-e '/CONFIG_MODULE_SIG/d'
-		-e '/CONFIG_SECURITY_LOCKDOWN/d'
-		-e '/CONFIG_KEXEC_SIG/d'
-		-e '/CONFIG_KEXEC_BZIMAGE_VERIFY_SIG/d'
-		-e '/CONFIG_SYSTEM_EXTRA_CERTIFICATE/d'
-		-e '/CONFIG_SIGNATURE/d'
-		# remove massive array of LSMs
-		-e 's/CONFIG_LSM=.*/CONFIG_LSM="yama"/'
-		-e 's/CONFIG_DEFAULT_SECURITY_SELINUX=y/CONFIG_DEFAULT_SECURITY_DAC=y/'
-		# nobody actually wants fips
-		-e '/CONFIG_CRYPTO_FIPS/d'
-		# these tests are really not necessary
-		-e 's/.*CONFIG_CRYPTO_MANAGER_DISABLE_TESTS.*/CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y/'
-		# probably not needed by anybody but developers
-		-e '/CONFIG_CRYPTO_STATS/d'
-		# 1000hz is excessive for laptops
-		-e 's/CONFIG_HZ_1000=y/CONFIG_HZ_300=y/'
-		# nobody is using this kernel on insane super computers
-		-e 's/CONFIG_NR_CPUS=.*/CONFIG_NR_CPUS=512/'
-		# we're not actually producing live patches for folks
-		-e 's/CONFIG_LIVEPATCH=y/CONFIG_LIVEPATCH=n/'
-		# this slows down networking in general
-		-e 's/CONFIG_IP_FIB_TRIE_STATS=y/CONFIG_IP_FIB_TRIE_STATS=n/'
-		# include font for normal and hidpi screens
-		-e 's/.*CONFIG_FONTS.*/CONFIG_FONTS=y\nCONFIG_FONT_8x16=y\nCONFIG_FONT_TER16x32=y/'
-		# we don't need to actually install system headers from this ebuild
-		-e '/CONFIG_HEADERS_INSTALL/d'
-		# enable /proc/config.gz, used by linux-info.eclass
-		-e '/CONFIG_IKCONFIG/s:.*:CONFIG_IKCONFIG=y\nCONFIG_IKCONFIG_PROC=y:'
+	local merge_configs=(
+		.config
+		"${FILESDIR}"/base.config
 	)
-	use debug || config_tweaks+=(
-		-e '/CONFIG_DEBUG_INFO/d'
-		-e '/CONFIG_DEBUG_RODATA_TEST/d'
-		-e '/CONFIG_DEBUG_VM/d'
-		-e '/CONFIG_DEBUG_SHIRQ/d'
-		-e '/CONFIG_DEBUG_LIST/d'
-		-e '/CONFIG_BUG_ON_DATA_CORRUPTION/d'
-		-e '/CONFIG_TORTURE_TEST/d'
-		-e '/CONFIG_BOOTTIME_TRACING/d'
-		-e '/CONFIG_RING_BUFFER_BENCHMARK/d'
-		-e '/CONFIG_X86_DECODER_SELFTEST/d'
-		-e '/CONFIG_KGDB/d'
-	)
-	sed -i "${config_tweaks[@]}" .config || die
+	use debug || merge_configs+=( "${FILESDIR}"/no-debug.config )
+
+	# support user configs
+	local shopt_save=$(shopt -p nullglob)
+	shopt -s nullglob
+	merge_configs+=( /etc/kernel/config.d/*.config )
```

Comment:
should not this path be `${EPREFIX}`ed?

Reply:
BROOT-ed probably.

Reply:
yeah `BROOT` sounds correct since it's build-time thing.